functions/pipelines.py

```diff
@@ -24,17 +24,73 @@
-                                             lst_num_experiments,
-                                             corrected,
-                                             use_pca,
-                                             num_PCs,
-                                             file_prefix,
-                                             input_file):
+def sample_level_simulation_uncorrected(run,
+                                        NN_architecture,
+                                        dataset_name,
+                                        analysis_name,
+                                        num_simulated_samples,
+                                        lst_num_experiments,
+                                        corrected,
+                                        use_pca,
+                                        num_PCs,
+                                        file_prefix,
+                                        input_file):
+  '''
+    This function performs runs series of scripts that performs the following steps:
+    1. Simulate gene expression data, ignorning the sample-experiment relationship
+    2. Add varying numbers of technical variation
+    3. Compare the similarity of the gene expression structure between the simulated data
+        vs simulated data + technical variation.  
+
+    Arguments
+    ----------
+    run: int
+        Unique core identifier that is used to create unique filenames for intermediate files
+
+    NN_architecture: str
+        Name of neural network architecture to use.
+        Format 'NN_<intermediate layer>_<latent layer>'
+
+    dataset_name: str
+        Either "Human_analysis" or "Pseudomonas_analysis"
+
+    analysis_name: str
+        Parent directory where simulated data with experiments/partitionings will be stored.
+        Format of the directory name is <dataset>_<sample/experiment>_lvl_sim 
+
+    num_simulated_samples: int
+        Number of samples to simulate
+
+    lst_num_experiments: list
+        List of different numbers of experiments that to add to
+        simulated data
+
+    corrected: bool
+        True if correction was applied
+
+    use_pca: bool
+        True if want to represent expression data in top PCs before
+        calculating similarity
+
+    num_PCs: int
+        Number of top PCs to use to represent expression data
+
+    file_prefix: str
+        File prefix to determine whether to use data before correction ("Experiment" or "Partition")
+        or after correction ("Experiment_corrected" or "Parition_corrected")
+
+    input_file: str
+        File name containing normalized gene expressiond data
+
+    Returns
+    --------
+    similarity_score_df: df
+        Similarity scores for each number of experiment/partition added per run
+
+    permuted_scre: df
+        Similarity score comparing the permuted data to the simulated data per run
+
+    '''
 
   # Input files
   local_dir = "/home/alexandra/Documents/"
```

Comment:
I always use the same directory, so since this parameter was static I just fixed it into each function.  But perhaps it does make sense to have the user pass this in as where they want to store intermediate data.  I can add this in the next PR